<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GUO-PC" name="Default test" tests="4" failures="0" timestamp="7 Dec 2017 07:28:29 GMT" time="0.328" errors="0">
  <testcase name="@BeforeClass initTest" time="0.0" classname="com.gorky.appiumdemo.testcases.login.LoginPage_001_LoginCase_Test">
    <failure type="java.lang.NumberFormatException" message="null">
      <![CDATA[java.lang.NumberFormatException: null
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.valueOf(Unknown Source)
	at com.gorky.appiumdemo.utils.SelectDriver.selectDriver(SelectDriver.java:114)
	at com.gorky.appiumdemo.base.BasePrepare.initTest(BasePrepare.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass initTest -->
  <testcase name="@AfterClass clenTest" time="0.0" classname="com.gorky.appiumdemo.testcases.login.LoginPage_001_LoginCase_Test">
    <skipped/>
  </testcase> <!-- @AfterClass clenTest -->
  <testcase name="@BeforeClass initTest" time="0.0" classname="com.gorky.appiumdemo.testcases.customer.CustomerPage_001_CustomerCase_Test">
    <failure type="java.lang.NumberFormatException" message="null">
      <![CDATA[java.lang.NumberFormatException: null
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.valueOf(Unknown Source)
	at com.gorky.appiumdemo.base.BasePrepare1.initTest(BasePrepare1.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass initTest -->
  <testcase name="@AfterClass clenTest" time="0.0" classname="com.gorky.appiumdemo.testcases.customer.CustomerPage_001_CustomerCase_Test">
    <skipped/>
  </testcase> <!-- @AfterClass clenTest -->
  <testcase name="@BeforeClass initTest" time="0.0" classname="com.gorky.appiumdemo.testcases.home.HomePage_001_HomeCase_Test">
    <skipped/>
  </testcase> <!-- @BeforeClass initTest -->
  <testcase name="@AfterClass clenTest" time="0.0" classname="com.gorky.appiumdemo.testcases.home.HomePage_001_HomeCase_Test">
    <skipped/>
  </testcase> <!-- @AfterClass clenTest -->
  <testcase name="loginDemo" time="0.003" classname="com.gorky.appiumdemo.testcases.login.LoginPage_001_LoginCase_Test">
    <skipped/>
  </testcase> <!-- loginDemo -->
  <testcase name="homeDemo" time="0.003" classname="com.gorky.appiumdemo.testcases.customer.CustomerPage_001_CustomerCase_Test">
    <skipped/>
  </testcase> <!-- homeDemo -->
  <testcase name="customerDemo" time="0.001" classname="com.gorky.appiumdemo.testcases.customer.CustomerPage_001_CustomerCase_Test">
    <skipped/>
  </testcase> <!-- customerDemo -->
  <testcase name="homeDemo" time="0.003" classname="com.gorky.appiumdemo.testcases.home.HomePage_001_HomeCase_Test">
    <skipped/>
  </testcase> <!-- homeDemo -->
</testsuite> <!-- Default test -->
